docs/runtimes/application.ipynb

```diff
@@ -12,6 +12,8 @@
     "The application runtime deploys a container image (for example, a web application) that is exposed on a specific port, and a command to run the HTTP server. The runtime is based on top of Nuclio, and adds the application as a side-car to a Nuclio function pod while the actual function is a reverse proxy to that application. \n",
     "\n",
     "You can set an existing image to run in the application, or let the application runtime build the side-car image for you, by specifying the source code, and pulling at build-time. \n",
+    "\n",
+    "> Note: The default base image is python:3.9 when not specifying and image for the application.\n",
```

Comment:
"> Note: The default base image is python 3.9 when not specifying and image for the application.\n",

Reply:
`python:3.9` is the name of the image. its a docker convention. I'll mark it as code to make it clear